web/react/components/admin_console/team_analytics.jsx

```diff
@@ -0,0 +1,353 @@
+// Copyright (c) 2015 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+var Client = require('../../utils/client.jsx');
+var Utils = require('../../utils/utils.jsx');
+var LineChart = require('./line_chart.jsx');
+
+export default class TeamAnalytics extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.getData = this.getData.bind(this);
+
+        this.state = {
+            users: null,
+            serverError: null,
+            channel_open_count: null,
+            channel_private_count: null,
+            post_count: null,
+            post_counts_day: null,
+            user_counts_with_posts_day: null,
+            recent_active_users: null,
+            newly_created_users: null
+        };
+    }
+
+    componentDidMount() {
+        this.getData(this.props.team.id);
+    }
+
+    getData(teamId) {
+        Client.getAnalytics(
+            teamId,
+            'standard',
```

Comment:
Not a big deal since this is the only place it's used but might be worth using a constant for here and below with 'post_counts_day' 0/5